packages/runtime/container-runtime/README.md

```diff
@@ -51,6 +51,54 @@ issue](https://github.com/microsoft/FluidFramework/wiki/Submitting-Bugs-and-Feat
 
 Thank you!
 
+## Data Virtualization For DataStores
+
+This is the ability of container to delay load datastores at any point in time by asking for the datastore snapshot at
+anytime, not caring about from where it is coming and then using that to load that datastore and return it to the app.
```

Comment:
This paragraph is hard to parse.
I'd say something like that
"It's a capability to exclude some content from initial snapshot (used when loading container) and thus improve boot performance of a container. Excluded content could be loaded at a later time when it's required."